### DISCUSSION TITLE
Web App Deployment

### DISCUSSION THREAD

**Discussion by codeit13** [iharshit009/SecretPeek#34] [Ideas]
I want to deploy the web app (Static Content) along with the API and Sockets all together, I wanted to know if is it a wise thing to do or not. Or like generally is it like sockets are implemented somewhere else.
I am new to this so sorry if this sounded like a vague question.

Also, I am thinking of deploying it in Heroku, I researched a bit and I think we can deploy these things there.

**Comment by iharshit009** [comment]
We would be deploying it in Heroku, I will share the credentials with you.
Yes we need to deploy the complete project, I would also share with you the
best way to do this. Currently sockets isn't implemented, so we would
deploy this as of now, later on when we would work on sockets, we would
modify it



[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
01/22/21,
07:31:33 PM
